keras_core/backend/tensorflow/core.py

```diff
@@ -163,5 +163,13 @@ def while_loop(
     )
 
 
+def fori_loop(lower, upper, body_fun, init_val):
```

Comment:
All backend functions should have the same signature across backends.

Reply:
this should be fixed as of most recent commit